aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -38,154 +38,6 @@ DEFINE_DISPATCH(argmin_stub);
 DEFINE_DISPATCH(cumsum_stub);
 DEFINE_DISPATCH(cumprod_stub);
 
-#define OPTION_TYPE_EQUALITY_CHECK(option, out, self) \
```

Comment:
All these are moved to `aten/src/ATen/native/ReduceOpsUtils.h` for visibility in CUDA.